spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -914,61 +990,64 @@ To replace the default tags, provide a `@Bean` that implements `RepositoryTagsPr
+Auto-configuration enables the instrumentation of all available RabbitMQ connection factories with a metric named `rabbitmq`.
 
 
 
 [[actuator.metrics.supported.spring-integration]]
 ==== Spring Integration Metrics
-Spring Integration provides {spring-integration-docs}system-management.html#micrometer-integration[Micrometer support] automatically whenever a `MeterRegistry` bean is available.
+Spring Integration automatically provides {spring-integration-docs}system-management.html#micrometer-integration[Micrometer support] whenever a `MeterRegistry` bean is available.
 Metrics are published under the `spring.integration.` meter name.
 
 
 
 [[actuator.metrics.supported.kafka]]
 ==== Kafka Metrics
-Auto-configuration will register a `MicrometerConsumerListener` and `MicrometerProducerListener` for the auto-configured consumer factory and producer factory respectively.
-It will also register a `KafkaStreamsMicrometerListener` for `StreamsBuilderFactoryBean`.
-For more details refer to {spring-kafka-docs}#micrometer-native[Micrometer Native Metrics] section of the Spring Kafka documentation.
+Auto-configuration registers a `MicrometerConsumerListener` and `MicrometerProducerListener` for the auto-configured consumer factory and producer factory, respectively.
+It alsos register a `KafkaStreamsMicrometerListener` for `StreamsBuilderFactoryBean`.
+For more detail, see the {spring-kafka-docs}#micrometer-native[Micrometer Native Metrics] section of the Spring Kafka documentation.
 
 
 
 [[actuator.metrics.supported.mongodb]]
 ==== MongoDB Metrics
-
+This section briefly describes the available metrics for MongoDB.
 
 
 [[actuator.metrics.supported.mongodb.command]]
-===== Command Metrics
-Auto-configuration will register a `MongoMetricsCommandListener` with the auto-configured `MongoClient`.
+===== MongoDB Command Metrics
+Auto-configuration registers a `MongoMetricsCommandListener` with the auto-configured `MongoClient`.
 
-A timer metric with the name `mongodb.driver.commands` is created for each command issued to the underlying MongoDB driver.
+A timer metric named `mongodb.driver.commands` is created for each command issued to the underlying MongoDB driver.
 Each metric is tagged with the following information by default:
 |===
 | Tag | Description
 
 | `command`
-| Name of the command issued
+| The name of the command issued.
 
 | `cluster.id`
-| Identifier of the cluster the command was sent to
+| The identifier of the cluster to which the command was sent.
 
 | `server.address`
-| Address of the server the command was sent to
+| The address of the server to which the command was sent.
 
 | `status`
-| Outcome of the command - one of (`SUCCESS`, `FAILED`)
+| The outcome of the command (`SUCCESS` or `FAILED`).
 |===
 
-To replace the default metric tags, define a `MongoCommandTagsProvider` bean, as shown in the following example:
+To replace the default metric tags, define a `MongoCommandTagsProvider` bean:
```

Comment:
Has the style changed here? Previous editing added "as shown in the following example" in situations like this.

Reply:
There's been discussion amongst the Tanzu writers about whether to have these "following" clauses. I favor them, but there's a case to be made for not using them (it's fewer words). I think about our English-as-a-second-language readers, though, who have largely been raised on a more formal kind of documentation. I'm a little conflicted about the whole issue, but I will stick to the more formal path going forward. Thanks for making me think about it again.